buildroot/bin/build_all_examples

```diff
@@ -35,28 +44,31 @@ else
   echo "Resuming from $2"
 fi
 
-TMPDIR=`mktemp -d`
+# Create a temporary folder if needed
+[[ -d $MFTEMP ]] || export MFTEMP=$(mktemp -d)
```

Comment:
I'm not very fan of polluting my environment.

Reply:
This doesn't work anyway unless you run with `source` or `.`.